### DISCUSSION TITLE
pandoc.exe: test.md: withBinaryFile: does not exist (No such file or directory)

### DISCUSSION THREAD

**Discussion by zk-zephyr** [jgm/pandoc#10016] [Q&A] [answered]
Hi Everyone,

I'm trying to follow one of the examples from the FAQ documentation: 
["How can I convert a whole directory of files from Markdown to RTF?"
](https://pandoc.org/faqs.html)

I also found (and commented on)[https://github.com/jgm/pandoc/issues/5429]( https://github.com/jgm/pandoc/issues/5429)

The command that I'm trying to run is:
`gci -r -i *.md |foreach{$rtf=$_.directoryname+"\"+$_.basename+".rtf"; pandoc -f markdown -s $_.name -o $rtf}`

I've also tried  `gci -r -i *.md |foreach{$docx=$_.directoryname+"\"+$_.basename+".docx";pandoc -f markdown+smart -s $_.name -o $docx}`

Yet I always get the error for all of the sub folders:

`pandoc.exe: test.md: withBinaryFile: does not exist (No such file or directory)`

The top level folder converts all the documents just fine. My project is small enough that I can manually move from folder to folder, but I think fixing the FAQ would be helpful. 

I've tried removing ` -s $_.name` and that causes pandoc to hang.

**Comment by kysko** [comment, marked as answer]
Use `$_.fullname` instead of `$_.name` for the full input file path.

**Reply by zk-zephyr** [reply]
Thanks so much!
